atd-vze/src/views/Crashes/PeopleDetailsCard.js

```diff
@@ -83,6 +92,7 @@ const PeopleDetailsCard = ({ isExpanded, toggleAccordion, ...props }) => {
             keyField={"primaryperson_id"}
             lookupOptions={lookupSelectOptions}
             mutation={primaryPersonMutation}
+            secondaryMutation={fatalityMutation}
```

Comment:
needed to add a secondary mutation that handles updates to `victim_name` bc this column exists in the fatalities table

Reply:
oh yeah, i really like that idea as well. i forgot that was even a possibility, im going to implement that! and yeah +1 @mddilley, it felt weird writing a `secondaryMutation` and im laughing thinking about there being a `quaternaryMutation` 😁